app/assets/javascripts/components/admin/unmatched_page.es6.jsx

```diff
@@ -0,0 +1,190 @@
+class UnmatchedPage extends React.Component {
+
+  constructor(props) {
+    super();
+    this.state = {
+      student_detail: false,
+      teacher_detail: false,
+      students: null,
+      teachers: null,
+      student: null,
+      teacher: null,
+    }
+
+    this.studentOnClick = this.studentOnClick.bind(this);
+    this.teacherOnClick = this.teacherOnClick.bind(this);
+    this.studentBackButton = this.studentBackButton.bind(this);
+    this.teacherBackButton = this.teacherBackButton.bind(this);
+    this.makeMatching = this.makeMatching.bind(this);
+  }
+
+  componentDidMount() {
+    this.loadStudents()
+  }
+
+  loadStudents() {
+    var studentRoute = ApiConstants.students.unmatched;
```

Comment:
just call this route automatically in Requester.get, don't have to set a variable